raft/server.hh

```diff
@@ -45,9 +45,14 @@ public:
         // Max size of a single command, add_entry with a bigger command will throw command_is_too_big_error.
         // The value of zero means no limit.
         size_t max_command_size = 0;
+
         // A callback to invoke if one of internal server
         // background activities has stopped because of an error.
         std::function<void(std::exception_ptr e)> on_background_error;
+
+        // The size of the applier fiber queue.
+        // This parameter is for testing only.
+        size_t applier_fiber_queue_size = 10;
```

Comment:
I do not like putting things for testing into the config. raft::server has a bunch of function intended to be used by tests only. Lest add a function to reconfigure the queue size there.